EIPS/eip-5313.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 5313
+title: Light Contract Ownership
+description: An interface for identifying ownership of contracts
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-5313-light-contract-ownership/10052
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-22
+---
+
+## Abstract
+
+This specification defines the minimum interface required to identify an account that controls a contract.
+
+## Motivation
+
+This is a slimmed-down alternative to [EIP-173](./eip-173).
```

Comment:
```suggestion
This is a slimmed-down alternative to [EIP-173](./eip-173.md).
```
FYI - you can accept changes from the GitHub interface. Just click "Commit Suggestion."